models/User.js

```diff
@@ -0,0 +1,47 @@
+const { Model, DataTypes } = require("sequelize");
+
```

Comment:
Need to add/require bcrypt:
`const bcrypt = require('bcrypt');`

Reply:
Added